client/src/components/mapping/TabColumnPicking/DynamicColumnPicker/index.tsx

```diff
@@ -61,10 +62,24 @@ const DynamicColumnPicker = ({ attribute, schema, source }: Props) => {
     undefined as number | undefined
   );
 
-  const [createAttribute] = useMutation(mCreateAttribute);
-  const [createSQLInput, { loading: creatingSQLInput }] = useMutation(
-    mCreateSQLInput
-  );
+  const onError = (error: ApolloError): void => {
```

Comment:
can't this be factorized ?

Reply:
I'd love to! I'm just not sure where to put it...